src/pages/index.ts

```diff
@@ -1,5 +1,8 @@
-import Task1 from "./task1/Task1";
-import Task2 from "./task2/Task2";
+import Home from './homePage/HomePage'
+import Task1 from './task1/Task1'
+import Task2 from './task2/Task2'
+import NotFound from './notFoundPage/NotFound'
+import PrivateRoute from './PrivateRoute'
+import SharedLayout from './sharedLayout/SharedLayout'
 
```

Comment:
You have two files that have the name "index.ts". I think it`s better to rename them

Reply:
I created these 'index.ts' files to export pages like this:
![image](https://user-images.githubusercontent.com/87831628/194067304-0db7337f-0b5e-49b9-943e-5f7ca45f28bc.png)

Reply:
I got it, but I meant that you have index.ts file in the Components directory and the file with the same name in Pages directory. So, I'm a bit confused by this because I would take one file for another)